.github/workflows/cdf_auth.yaml

```diff
@@ -16,11 +16,11 @@ jobs:
     container:
       image: cognite/toolkit:0.4.9
       env:
-        CDF_CLUSTER: ${{ vars.CDF_CLUSTER }}
```

Comment:
I think you need to remove the `environment: CD` above.

Reply:
why?

Reply:
It looks like you are not using GitHub environment?

Reply:
Ah right, no just using repo variables.